File: src/main/java/org/azamov/learnjakarta/task2/java/session/AttributeListener.java
package org.azamov.learnjakarta.task2.java.session;

import javax.servlet.annotation.WebListener;
import javax.servlet.http.HttpSessionAttributeListener;
import javax.servlet.http.HttpSessionBindingEvent;

@WebListener
public class AttributeListener implements HttpSessionAttributeListener {
    @Override
    public void attributeAdded(HttpSessionBindingEvent event) {
    }

    @Override
    public void attributeRemoved(HttpSessionBindingEvent event) {
    }

    @Override
    public void attributeReplaced(HttpSessionBindingEvent event) {
    }
}

File: src/main/java/org/azamov/learnjakarta/task2/kt/session/AttributeListener.kt
package org.azamov.learnjakarta.task2.kt.session

import javax.servlet.annotation.WebListener
import javax.servlet.http.HttpSessionAttributeListener
import javax.servlet.http.HttpSessionBindingEvent

@WebListener
class AttributeListener : HttpSessionAttributeListener {

    override fun attributeAdded(event: HttpSessionBindingEvent?) {
        super.attributeAdded(event)
    }

    override fun attributeRemoved(event: HttpSessionBindingEvent?) {
        super.attributeRemoved(event)
    }

    override fun attributeReplaced(event: HttpSessionBindingEvent?) {
        super.attributeReplaced(event)
    }
}